pkg/outposts/cluster_extender.go

```diff
@@ -31,8 +31,9 @@ type stackUpdater interface {
 	AppendNewClusterStackResource(ctx context.Context, extendForOutposts, plan bool) (bool, error)
 }
 
-//counterfeiter:generate -o fakes . ClusterToExtend
 // ClusterToExtend represents a cluster that needs to be extended.
+//
```

Comment:
Can you pls remove changes like these from the PR?

Reply:
thx! fixed.